compiler/luci/pass/include/luci/Pass/QuantizeWithMinMaxPass.h

```diff
@@ -62,6 +77,7 @@ class QuantizeWithMinMaxPass : public logo::Pass
   void set_output_type(loco::Graph *graph) const;
 
 private:
+  std::unique_ptr<Context> _ctx;
```

Comment:
Q) maybe just `Context _ctx;`? need to use unique ptr?

Reply:
I chose to use `unique_ptr` to avoid copy overhead.

For now, copy is not burdensome. But it can be if we pass information of many layers (ex: precision of each layer).